releasenotes/notes/fix-callback-multi-circuits-f8444aef40c7bd89.yaml

```diff
@@ -0,0 +1,8 @@
+---
+fixes:
+  - |
+    Fixed a bug in ``PassManager.run()`` where the ``callback`` function was not
+    invoked when running with multiple circuits. The callback is now correctly
+    triggered for each pass on each circuit, including in parallel execution.
+    The fix is validated through a unit test that also demonstrates the use of
```

Comment:
I suggest removing the last sentence as it describes the test used to validate the fix, rather than fix itself.